publish/page_template.md

```diff
@@ -2,15 +2,33 @@
 title: {{issue.title}} - Python Community News {{issue.episode_date}}
 slug: "{{issue.episode_date}}"
 date: {{issue._created_at}}
-podcast: {{issue.podcast}}
-youtube: {{issue.youtube}}
+{% if issue['podcast'] %}podcast: {{issue.podcast}}
+{% endif %}{% if issue['youtube'] %}youtube: {{issue.youtube}}{% endif %}
```

Comment:
Can we reformat this so that the podcast and youtube attributes are isolated on their own lines?

Reply:
Doing so actually adds empty lines to the template itself which may have unintended consequences with Render Engine.


Previously there was a manual break check for Render Engine. That was replaced with the frontmatter package, but I'm not sure how `python-frontmatter` will handle the empty lines.